builder/src/firmware.rs

```diff
@@ -5,6 +5,16 @@
 
 use crate::FwId;
 
+pub fn rom() -> &'static FwId<'static> {
```

Comment:
nit: maybe call this something like `rom_from_env`? Makes it more clear in the test code how the ROM image is being configured.

Reply:
Done.